uint8 id
bool enable 
bool enable_distortion
float32 distortion 
---
bool ack
